batavia/modules/time.js

```diff
@@ -156,15 +156,17 @@ time.mktime = function(sequence) {
         ],
         tz_name
     )
-    var d = m.toDate()
+    var d 
+    d = m.toDate()
 
     if (isNaN(d)) {
         // d is too large per ECMA specs
         // source: http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
```

Comment:
At column 10: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)